ansible/roles/kafka/tasks/install.yml

```diff
@@ -5,4 +5,6 @@
   changed_when: false
 
 - name: Fetch kafka binary package
-  get_url: dest="{{ kafka_tarball_location }}/{{ kafka_tarball_name }}" url="{{apache_mirror.stdout}}/kafka/{{kafka_version}}/{{kafka_tarball_name}}"
+  get_url:
+    dest: "{{ kafka_tarball_location }}/{{ kafka_tarball_name }}"
+    url: "https://archive.apache.org/dist/kafka/{{kafka_version}}/{{kafka_tarball_name}}"
```

Comment:
Temporary - kafka 2.6.0 was moved to archive, after update we can use {{apache_mirror.stdout}} back as it selects proper apache mirror.